biogeochem/EDCanopyStructureMod.F90

```diff
@@ -1041,9 +1104,9 @@ subroutine canopy_summarization( nsites, sites, bc_in )
                       currentCohort%pft,currentCohort%canopy_trim
                 call endrun(msg=errMsg(sourcefile, __LINE__))
              endif
-             if( (currentCohort%bsw + currentCohort%bl + currentCohort%br) <= 0._r8)then
+             if( (sapw_c + leaf_c + fnrt_c) <= 0._r8)then
                 write(fates_log(),*) 'FATES: alive biomass is zero in canopy_summarization', &
```

Comment:
Just to be clear, we are keeping "live" and "alive" BUT getting rid of "dead" for "structural" correct?  so in this case we will have live sap, lead, fine root carbon which then could turn to structural

Reply:
the six prognostic carbon pools used in the current hypothesis (and the next) are leaf, fine-root, sapwood, structure, storage and reproduction.  Each of which is exclusive of the others (none overlap).  Everything except "structure" should be considered alive (as far as I know), because they all respire to some extent right?

We don't have an "alive" or "live" prognostic pool, as it is the sum of the non-structural pools, except when we are writing diagnostics.

So, as for terminology, I think it seems ok to keep "live" and "alive" when we are working with diagnostic variables..

Reply:
OK thanks @rgknox very helpful!